src/views/MatchDetail.vue

```diff
@@ -272,12 +285,45 @@ export default class MatchDetailView extends Vue {
   }
 
   get scoresOfWinners() {
-    const scoresOfWinners = this.playerScores.filter(
-      (s) =>
-        this.match.teams[0].players.some((player) =>
+    const winningTeam = this.match.teams[0];
+    let scoresOfWinners = this.playerScores
+      .filter((s) =>
+        winningTeam.players.some((player) =>
           player.battleTag.startsWith(s.battleTag)
         )
-    );
+      )
+      .map((s) => {
```

Comment:
Can you extract this into a function with some parameters cuz it seems like alot of duplicated code:
full/partial battleTag match, winner/loser

Reply:
@abeninski PR has been updated.

Reply:
Oh much better